.github/workflows/ci.yml

```diff
@@ -21,20 +21,19 @@ concurrency:
   group: '${{ github.workflow }}-${{ github.head_ref || github.run_id }}'
   cancel-in-progress: true
 env:
-  STACKFLAGS: '--pedantic --stack-root ${{ github.workspace }}/.stack'
+  SKIP: ormolu
```

Comment:
Will the `--pedantic` flag still be passed to the Juvix build? I think this was useful to have.

Reply:
It's true by default using the frakle action.
https://github.com/freckle/stack-action

Reply:
Perfect!